apps/static/dance/ai/model/cached_palettes_map.json

```diff
@@ -1,109 +1,107 @@
+      0.329, 0.238, 0.268, 0.194, 0.177, 0.355, 0.123, 0.211, 0.319, 0.141,
+      0.321, 0.268, 0.211, 0.279, 0.303, 0.246, 0.198, 0.466, 0.212, 0.383,
+      0.743
     ],
     "artist-palette": [
-      0.769, 0.804, 0.777, 0.758, 0.811, 0.82, 0.845, 0.788, 0.76, 0.803, 0.791,
-      0.765, 0.756, 0.783, 0.757, 0.769, 0.774, 0.77, 0.775, 0.788, 0.787
+      0.275, 0.365, 0.207, 0.173, 0.496, 0.573, 0.649, 0.333, 0.194, 0.362,
+      0.268, 0.287, 0.298, 0.26, 0.329, 0.345, 0.254, 0.243, 0.23, 0.291, 0.365
     ],
     "mirror-ball": [
-      0.739, 0.785, 0.762, 0.782, 0.813, 0.771, 0.791, 0.8, 0.73, 0.794, 0.744,
-      0.761, 0.753, 0.754, 0.75, 0.76, 0.763, 0.753, 0.797, 0.758, 0.763
+      0.029, 0.188, 0.177, 0.161, 0.17, 0.254, 0.238, 0.177, 0.039, 0.282, 0.07,
+      0.11, 0.073, 0.083, 0.096, 0.139, 0.158, 0.05, 0.275, 0.094, 0.125
     ],
     "musical-note": [
-      0.752, 0.749, 0.769, 0.772, 0.752, 0.727, 0.735, 0.756, 0.727, 0.752,
-      0.747, 0.757, 0.725, 0.747, 0.724, 0.731, 0.749, 0.745, 0.754, 0.729,
-      0.725
+      0.14, 0.031, 0.127, 0.297, 0.122, 0.099, 0.088, 0.097, 0.109, 0.111, 0.08,
+      0.137, 0.038, 0.072, 0.088, 0.07, 0.099, 0.058, 0.184, 0.056, 0.068
     ],
-    "cold-face": [
-      0.79, 0.778, 0.804, 0.755, 0.775, 0.798, 0.769, 0.774, 0.784, 0.782,
-      0.788, 0.775, 0.749, 0.795, 0.776, 0.773, 0.787, 0.815, 0.772, 0.81, 0.881
+    "pile-of-poo": [
+      0.105, 0.124, 0.439, 0.142, 0.086, 0.146, 0.167, 0.139, 0.193, 0.168,
+      0.083, 0.13, 0.08, 0.139, 0.134, 0.13, 0.219, 0.155, 0.322, 0.168, 0.09
     ],
     "rocket": [
-      0.74, 0.759, 0.778, 0.755, 0.774, 0.767, 0.8, 0.783, 0.739, 0.795, 0.774,
-      0.805, 0.745, 0.761, 0.782, 0.784, 0.782, 0.773, 0.789, 0.775, 0.773
+      0.065, 0.066, 0.099, 0.089, 0.145, 0.131, 0.169, 0.164, 0.085, 0.162,
+      0.088, 0.314, 0.042, 0.143, 0.217, 0.203, 0.11, 0.122, 0.163, 0.136, 0.155
     ],
     "face-savoring-food": [
-      0.774, 0.778, 0.788, 0.768, 0.777, 0.803, 0.781, 0.772, 0.78, 0.763,
-      0.769, 0.766, 0.763, 0.782, 0.779, 0.789, 0.829, 0.786, 0.793, 0.79, 0.752
+      0.27, 0.202, 0.137, 0.153, 0.177, 0.274, 0.152, 0.142, 0.185, 0.157,
+      0.194, 0.144, 0.262, 0.255, 0.218, 0.226, 0.39, 0.274, 0.189, 0.322, 0.188
     ],
     "fire": [
-      0.791, 0.787, 0.812, 0.767, 0.782, 0.781, 0.793, 0.81, 0.78, 0.801, 0.818,
-      0.859, 0.775, 0.831, 0.818, 0.83, 0.794, 0.871, 0.801, 0.881, 0.804
+      0.214, 0.245, 0.261, 0.148, 0.218, 0.267, 0.256, 0.267, 0.242, 0.275,
+      0.22, 0.509, 0.213, 0.412, 0.464, 0.435, 0.215, 0.666, 0.271, 0.543, 0.333
     ],
-    "soft-ice-cream": [
-      0.779, 0.766, 0.795, 0.76, 0.757, 0.887, 0.813, 0.773, 0.764, 0.789,
-      0.794, 0.779, 0.776, 0.83, 0.789, 0.798, 0.856, 0.787, 0.782, 0.796, 0.808
+    "grinning-face-with-big-eyes": [
+      0.197, 0.368, 0.304, 0.164, 0.329, 0.316, 0.268, 0.251, 0.177, 0.333,
+      0.145, 0.326, 0.239, 0.335, 0.354, 0.345, 0.38, 0.24, 0.42, 0.279, 0.293
```

Comment:
Is it possible to output these in a consistent order?  It would make it a bit easier to confirm that nothing is breaking due to changing the set of keys.

Reply:
I'll have it alphabetize the outputs to help just make it easier to confirm with a visual scan

Reply:
And we discuss offline already, but if you could share a reference to the generating script, that would be helpful. Thanks so much and kudos for taking on yet another model change!